### DISCUSSION TITLE
Title column in the middle of a grid

### DISCUSSION THREAD

**Discussion by vozvik** [mar10/wunderbaum#11] [General]
Hi Martin,

I found some peculiar behaviour if the title column (id *) is not at the leftmost position in the grid:
https://codepen.io/vozvik/pen/bGKaQGV

Resizing grid no longer seems to work and the tree behaviour seems to be also off. Can you pease check if I am doing something wrong? Thank you!

**Comment by vozvik** [comment]
And I guess this could even be an invalid use-case for actual treegrids, on the other hand, I stumbled upon this when using Wunderbaum to show a simple grid and I can also introduce a workaround by renaming/reshuffling columns.

**Comment by mar10** [comment]
Hi Viktor,
yes, as for now, WB only supports the title in the leftmost column (also the fixed column mode only handles this)
I will add a check and erro rmessage to make this more clear,
